src/main.ts

```diff
@@ -89,6 +89,21 @@ async function run() {
       console.log(`${script}`);
     });
 
+    // android-home
+    const androidHome = core.getInput('android-home');
+    if (androidHome) {
+      core.exportVariable('ANDROID_HOME', `${process.env.HOME}/android-sdk`);
+      core.exportVariable('ANDROID_SDK_ROOT', `${process.env.HOME}/android-sdk`);
+    } else {
+      // If this is a self-hosted environment, and the android-home isn't set,
+      // then use a default path in user's home.
+      const selfHosted = core.getInput('self-hosted');
+      if (selfHosted) {
+        core.exportVariable('ANDROID_HOME', `${process.env.HOME}/android-sdk`);
+        core.exportVariable('ANDROID_SDK_ROOT', `${process.env.HOME}/android-sdk`);
+      }
+    }
```

Comment:
We don't need this else branch if we always do a fresh install / configuration at `${process.env.HOME}/android-sdk` when `fresh-sdk-installation` is true 😃